File: task3/Javascript/Subtask2.js
const fs = require('fs');

// Read the string from input.txt
fs.readFile('input.txt', 'utf8', (err, data) => {
    if (err) {
        console.error('Error reading the file:', err);
        return;
    }

    // Write the string to output.txt
    fs.writeFile('output.txt', data, (err) => {
        if (err) {
            console.error('Error writing to the file:', err);
        } else {
            console.log('File has been copied successfully.');
        }
    });
});

File: task3/GO/Subtask2.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	// Open the input file for reading
	inputFile, err := os.Open("input.txt")
	if err != nil {
		fmt.Println("Error opening input file:", err)
		return
	}
	defer inputFile.Close()

	// Open the output file for writing
	outputFile, err := os.Create("output.txt")
	if err != nil {
		fmt.Println("Error creating output file:", err)
		return
	}
	defer outputFile.Close()

	// Read from the input file
	scanner := bufio.NewScanner(inputFile)
	for scanner.Scan() {
		line := scanner.Text()
		// Write to the output file
		_, err := outputFile.WriteString(line + "\n")
		if err != nil {
			fmt.Println("Error writing to output file:", err)
			return
		}
	}

	if err := scanner.Err(); err != nil {
		fmt.Println("Error reading from input file:", err)
	}
}
